src/main/java/org/synyx/urlaubsverwaltung/person/basedata/PersonBasedataService.java

```diff
@@ -0,0 +1,35 @@
+package org.synyx.urlaubsverwaltung.person.basedata;
+
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+import static org.synyx.urlaubsverwaltung.person.basedata.PersonBasedataMapper.mapFromEntity;
+import static org.synyx.urlaubsverwaltung.person.basedata.PersonBasedataMapper.mapToEntity;
+
+@Service
+public class PersonBasedataService {
```

Comment:
Is this service used in other packages? If yes we need an interface if not please close it to package private

Reply:
added